src/world/common/enemy/ClubbaNappingAI.inc.c

```diff
@@ -0,0 +1,318 @@
+#include "common.h"
+
+extern Npc* wPartnerNpc;
```

Comment:
this extern shouldn't be needed (is causing unnecessary warnings). Theoretically this should be defined in npc.h (which should be included anyway, since you're using npc structs and functions)